manual/conf.py

```diff
@@ -42,4 +42,11 @@
 intersphinx_mapping = {
     "python": ("https://docs.python.org/3/", None),
     "pytket": (pytketdoc_base, None),
+    "qiskit": ("https://docs.quantum.ibm.com/api/qiskit", None),
+    "pytket-qiskit": ("https://tket.quantinuum.com/extensions/pytket-qiskit/", None),
+    "pytket-quantinuum": (
+        "https://tket.quantinuum.com/extensions/pytket-quantinuum/",
+        None,
+    ),
+    "sympy": ("https://docs.sympy.org/latest/index.html", None),
```

Comment:
Still can't really seem to link to sympy stuff in a nice way

Reply:
I think you can get the sympy object inventory at https://docs.sympy.org/latest/objects.inv, so that might help with figuring out what it actually calls its linkable items? (No worries if you'd already done this, but in case you hadn't, maybe it could help.)

Reply:
I had a look at this again, and found a useful tool for messing around with objects.inv files: https://sphobjinv.readthedocs.io/en/latest/cli/index.html

What I did with it was:

```
sphobjinv convert plain <input objects.inv file> output.txt
```

to get a plain text version of objects.inv, which you can then look at to figure out what you need to link to. But it also has a `sphobjinv suggest` subcommand that takes the name of something you want to link to and searches for keys that have that name.

Can help you out with the links to this, if that's useful?